features/proxy_config.feature

```diff
@@ -334,7 +334,18 @@ Feature: Proxy configuration
         """
         \"http://wronguser:wrongpassword@.*:3128\" is not working. Not setting as proxy.
         """
-    
+        When I run `pro config set http_proxy=http://someuser:somepassword@$behave_var{machine-ip proxy}:3128` with sudo
+        When I run `pro config show` with sudo
+        Then stdout matches regexp:
+        """
+        http_proxy              http://someuser:somepassword@$behave_var{machine-ip proxy}:3128
+        """
+        When I run `pro config show` as non-root
+        Then stdout matches regexp:
+        """
+        http_proxy              <REDACTED>
```

Comment:
```suggestion
        http_proxy              http://<REDACTED>:<REDACTED>@$behave_var{machine-ip proxy}:3128
```

Reply:
@dheyay I changed my mind. The urlparse function in the stdlib is not nice enough to make it easy to reconstruct the url with redacted parts, and it's not worth the risk of a bug here to try to do it ourselves via regex or otherwise. Redacting the whole proxy url is good.